packages/graph/hash_graph/lib/graph/src/store/postgres/query/condition.rs

```diff
@@ -261,8 +261,8 @@ mod tests {
                     Cow::Borrowed("right"),
```

Comment:
nit for line 254, name of test doesn't make sense anymore `render_without_parameters` :D

Reply:
good spot, fixed the name of the test in https://github.com/hashintel/hash/pull/1230/commits/e14d3f1ed81598c10bb5a9197f91be5ac70606c8